app/error-templates/corrections/DR9876-Length.html

```diff
@@ -0,0 +1,3 @@
+<div class="CorrectionLength">
+    <p>Enter a maximum of 10 characters for the SIMPLE FIELD</p>
```

Comment:
Ok - yep my bad. I put this in when I was doing the documentation of how to add an error snippet and its found its way into the branch. I will delete it.